crates/sncast/src/starknet_commands/deploy.rs

```diff
@@ -46,19 +57,113 @@ pub struct Deploy {
     pub version: Option<DeployVersion>,
 }
 
+impl Deploy {
+    pub fn build_artifacts(
```

Comment:
This function does two things, builds artifacts and gets a specified contract from artifacts